cmd/ssh-auth-none-demo/ssh-auth-none-demo.go

```diff
@@ -62,10 +62,14 @@ func main() {
 		Version: "Tailscale",
 		Handler: handleSessionPostSSHAuth,
 		ServerConfigCallback: func(ctx ssh.Context) *gossh.ServerConfig {
+			start := time.Now()
 			return &gossh.ServerConfig{
 				ImplicitAuthMethod: "tailscale",
 				NoClientAuth:       true, // required for the NoClientAuthCallback to run
-				NoClientAuthCallback: func(gossh.ConnMetadata) (*gossh.Permissions, error) {
+				NoClientAuthCallback: func(cm gossh.ConnMetadata) (*gossh.Permissions, error) {
+					cm.SendAuthBanner(fmt.Sprintf("# Banner: doing none auth at %v\r\n", time.Since(start)))
+					<-time.After(time.Second)
+					cm.SendAuthBanner(fmt.Sprintf("# Banner: access granted at %v\r\n", time.Since(start)))
```

Comment:
Say "Banner2"?

And, if the username is "banners" can you make it count to 5 or 10 instead, showing bannerN N times 1 sec apart?

Reply:
I think I understood what you said, PTAL?